test/cmdlineTests.sh

```diff
@@ -133,37 +159,44 @@ function test_solc_behaviour()
     if [[ $exitCode -ne "$exit_code_expected" ]]
     then
         printError "Incorrect exit code. Expected $exit_code_expected but got $exitCode."
-        rm -f $stdout_path $stderr_path
         exit 1
     fi
 
     if [[ "$(cat $stdout_path)" != "${stdout_expected}" ]]
     then
         printError "Incorrect output on stdout received. Expected:"
-        echo -e "${stdout_expected}"
+        echo "${stdout_expected}"
```

Comment:
I think it does format some json output a bit more nicely, if I remember correctly.